content/modules/redisearch/release-notes/redisearch-2.0-release-notes.md

```diff
@@ -0,0 +1,71 @@
+
+### Details
+* When you create an index, you must specify a prefix condition and/or a filter. This determines which hashes RediSearch will index.
+* Several RediSearch commands now map to their Redis equivalents: `FT.ADD` -> `HSET`, `FT.DEL` -> `DEL` (DD by default), `FT.GET` -> `HGETALL`, `FT.MGET` -> `HGETALL`.
+* RediSearch indexes no longer reside within the key space, and the indexes are no longer saved to the RDB.
+* You can [upgrade from RediSearch 1.x](https://oss.redislabs.com/redisearch/master/Upgrade_to_2.0/).
+
+### Noteworthy changes
+  - [#1246](https://github.com/RediSearch/RediSearch/pull/1246): [`  geodistance`](https://oss.redislabs.com/redisearch/master/Aggregations/#list_of_geo_apply_functions) function for `FT.AGGREGATE` APPLY operation.
+  - [#1394](https://github.com/RediSearch/RediSearch/pull/1394): Expired documents (TTL) will be removed from the index.
+  - [#1394](https://github.com/RediSearch/RediSearch/pull/1394): [Optimization](https://oss.redislabs.com/redisearch/master/Configuring/#partial_indexed_docs) to avoid reindexing documents when non-indexed fields are updated.
+  - After [index creation](https://oss.redislabs.com/redisearch/Commands/#ftcreate), an initial scan starts for existing documents.  You can check the status of this scan by calling [`FT.INFO`](https://oss.redislabs.com/redisearch/Commands/#ftinfo) and looking at the `indexing` and `percent_indexed` values.  While `indexing` is true, queries return partial results.
+  - [#1435](https://github.com/RediSearch/RediSearch/pull/1435): `NOINITIALINDEX` flag on [`FT.CREATE`](https://oss.redislabs.com/redisearch/Commands/#ftcreate) to skip the initial scan of documents on index creation.
+  - [#1401](https://github.com/RediSearch/RediSearch/pull/1401): Support upgrade from v1.x and for reading RDB's created by RediSearch 1.x ([more information](https://oss.redislabs.com/redisearch/master/Upgrade_to_2.0/)).
+  - [#1445](https://github.com/RediSearch/RediSearch/pull/1445): Support for load event. This event indexes documents when they are loaded from RDB, ensuring that indexes are fully available when RDB loading is complete (available from Redis 6.0.7 and above).
+  - [#1384](https://github.com/RediSearch/RediSearch/pull/1384): [`FT.DROPINDEX`](https://oss.redislabs.com/redisearch/Commands/#ftdropindex) which by default does not delete the documents (see deprecated `FT.DROP`).
+  - [#1385](https://github.com/RediSearch/RediSearch/pull/1385): Add index definition to [`FT.INFO`](https://oss.redislabs.com/redisearch/master/Commands/#ftinfo) response.
+   - [#1097](https://github.com/RediSearch/RediSearch/pull/1097): Add Hindi snowball stemmer.
+   - The `FT._LIST` command allows to query all available indices.  Note that this is a temporary command as indicated by the `_` The final solution will be a [`SCAN`](https://redis.io/commands/scan)-like command for databases with many indices.  This command is for this reason not documented.
```

Comment:
@K-Jo Why even include this in the release notes if we're not documenting it?

Reply:
That's a good question.  I'm in doubt that maybe I should document it.  Actually I start to believe we should.  I'm just afraid too many downstream applications will rely on the command.  I want it to be used for now by tools like RedisInsight and developers trying to figure out their index names. https://github.com/RediSearch/RediSearch/pull/1337 let's leave as is for now